### ISSUE TITLE
ISSUE: New variant of UPS pending delivery subject line

### ISSUE THREAD

**Issue by iankaufmann** [moralmunky/Home-Assistant-Mail-And-Packages#592]
**Describe the bug**
Similar to (https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/issues/587) I noticed that my UPS deliveries today had the subject line:
"UPS Pre-Arrival: Your Driver is Arriving Soon! Follow on a Live Map"

**Environment (please complete the following information):**
 - OS: HassOS
 - Type of device: NUC
 - Home Assistant version: core-2021.11.5
 - Component version: 0.3.4-b11

**Additional context**
Opening up const.py and adding to "ups_delivering" -> "subject" array and then restarting HA fixed it for me.  Just thought I'd pass it along.

**Comment by firstof9** [comment]
Did the delivered email change or is it the same?

**Comment by iankaufmann** [comment]
I haven't received it yet, even though it is delivered (about a half hour ago).  It was Amazon via UPS and I got the Amazon delivered email.  I'll update you here when I get the delivered message!  Thanks!!

**Comment by firstof9** [comment]
> It was Amazon via UPS

Prime example of why we can't really utilize the amazon sensors in the overall package sensors. 😞

**Comment by iankaufmann** [comment]
Yeah that makes sense.  The UPS emails are always a little more delayed than Amazon's, so we'll see if we get a delivery notice from UPS.  UPS sent the shipment and delivery today notifications, so I don't see why they wouldn't also send the delivered one eventually, despite the order being from Amazon.  Nothing yet though...

**Comment by iankaufmann** [comment]
Well, I never got anything.  I just double checked that my ups.com preferences were set correctly and they are.  Not sure why I didn't get it but after 4 hours I assume it's not coming.  I'll have to test it out next time I get something from UPS.  Hopefully nothing has changed with the delivered subject.
